src/test/java/org/junitpioneer/jupiter/AbstractEntryBasedExtensionTest.java

```diff
@@ -38,6 +38,7 @@ void setUp() {
 	}
 
 	@Test
+	@Issue("432")
```

Comment:
<3

Reply:
Wonder why we didn't use it before, so helpful! 👏